package com.semanticsquare.exceptions;

import java.io.FileNotFoundException;

public class ExceptionsDemo {
	
	public static void main(String[] args) {	
		System.out.println("\nInside main ...");		
		//try {
			share();
		/*} catch (FileNotFoundException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}	*/				
		System.out.println("\nEnd of main ...");
	}

	private static void share() {
		System.out.println("\nInside share ...");
		// HttpConnect.send(0, "hello", "http://www.goodsnips.com");
		try {
			HttpConnect.send(0, "hello", "http://www.goodsnips.com");
		} catch (FileNotFoundException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}		
		
		System.out.println("\nEnd of share ...");
	}	
	 
}